### DISCUSSION TITLE
使用alist+rclone由google drive上传至115过程遇到的问题

### DISCUSSION THREAD

**Discussion by ice0625** [AlistGo/alist#4941] [General]
由于115上传实在太慢,由其在国外上传更慢,小弟忽发其想,如使用alist以webdav方式挂载google drive及115,以rclone由google drive复制档案到115,上传至115的速度有明显改善,小弟用的是1Gpbs,速度差不多可以到50%~70%,与以115官方浏览器或官方软件差多了(官方的上传速度顶多就0.4Mpbs左右,极度慢和不稳定,可是却碰到很大一个问题,就是不管以cookie或二维码令牌登录115,以rclone上传115一段时间后(约数分钟至数小时),必定会出现这个错误:  ERROR : xxxx.mp4: Failed to copy: Locked: 423 Locked,上传速度会降至数百kb什至无法上传,换新的cookie或二维码令牌可以重新上传,但过不久又再出现一样的问题,请问板里各位大神有否遇到过相同问题?这有无方法可以改善或解决?(当然,这有可能把115弄至封号也说不定?)

**Comment by gdtv** [comment]
alist+rclone+移动云盘也是这样，上传大文件会 “Failed to copy: Locked: 423 Locked”，但是在alist的web ui里上传又没问题。

**Reply by Jiayou-Chao** [reply]
在rclone里设置 `--timeout 0` 试试，alist需要先缓存，如果上传速度又很慢的话会让rclone 错误的认为是IO idle timeout
